docs/en/api/FS-API.md

```diff
@@ -187,23 +212,66 @@ Users can override the default policy class in the
 [configuration file]({{ '/en/operation/Configuration.html' | relativize_url }}) at property
 `alluxio.user.block.write.location.policy.class`. The built-in policies include:
 
-* **LocalFirstPolicy (alluxio.client.block.policy.LocalFirstPolicy)**
+* [LocalFirstPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstPolicy.html)
+
+  **This is the default policy.**
+
+  > A policy that returns the local worker first, and if the local worker doesn't
+  > exist or have enough availability, will select the nearest worker from the active
+  > workers list with sufficient availability.
+  >
+  > The calculation of which worker gets selected is done for each block write.
+
+* [LocalFirstAvoidEvictionPolicy](https://docs.alluxio.io/os/javadoc/{{site.ALLUXIO_MAJOR_VERSION}}/alluxio/client/block/policy/LocalFirstAvoidEvictionPolicy.html)
+
+  This is the same as `LocalFirstPolicy` with the following addition:
 
-    Returns the local worker first, and if it does not have enough capacity of a block,
-    randomly picks a worker from the active workers list. This is the default policy.
+  > A policy that returns the local worker first, and if the local worker doesn't
+  > exist or have enough availability, will select the nearest worker from the active
```

Comment:
You might wanna add definition of "nearest" by pointing to the tiered identity doc page.